hints/hints.go

```diff
@@ -0,0 +1,76 @@
+package hints
+
+import (
+	"math/rand"
+	"time"
+
+	"github.com/manifoldco/torus-cli/ui"
+)
+
+var commandHints map[string][]string
+
+// Init generates the commandHints map
+func Init() {
+	commandHints = map[string][]string{
+		"allow": {
+			"Grant additional access to a team or role using `torus allow`",
+		},
+		"context": {
+			"Your linked organization and project are found in .torus.json after running `torus link`",
+		},
+		"deny": {
+			"Restrict access for a team or role using `torus deny`",
```

Comment:
We need the hints to remain succinct, they're not a place for teaching concepts but rather leading the users to the concepts they need to understand.

Without computing account context with every command you're going to have some that are one or two steps away from being applicable, but they're still relevant to learning.